tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html

```diff
@@ -113,6 +113,7 @@
         height: auto;
         position: relative;
         margin: 0 15px 40px 0;
+        overflow-y: auto;
```

Comment:
Actually, thanks for pointing it out, this should be `overflow-x`. In y-axis, the page grows infinitely so there should be no scrollbar. If I am understanding your screenshots correctly, you probably had `overflow-x`.